rainforest-cli.go

```diff
@@ -125,10 +125,11 @@ func main() {
 
 	app.Commands = []cli.Command{
 		{
-			Name:    "run",
-			Aliases: []string{"r"},
-			Usage:   "Run your tests on Rainforest",
-			Action:  startRun,
+			Name:         "run",
+			Aliases:      []string{"r"},
+			Usage:        "Run your tests on Rainforest",
+			OnUsageError: onCommandUsageErrorHandler("run"),
+			Action:       startRun,
 			Description: "Runs your tests on Rainforest platform. " +
 				"You need to specify list of test IDs to run or use keyword 'all'. " +
```

Comment:
Is this indentation legit? It looks odd that the string for the `Description` field doesn't start in the same column as the other values.

Reply:
I agree that it looks weird/inconsistent but it's what gofmt gives me :man_shrugging: